src/collections/_documentation/apm/distributed-tracing.md

```diff
@@ -0,0 +1,344 @@
+```
+
+**Managing Transactions**
+
+Let’s say you want to create a [transaction]({%- link _documentation/apm/apm-glossary.md -%}#transaction) for an expensive operation (for example, `process_item`) and send the result to Sentry:
+
+```python
+import sentry_sdk
+
+while True:
+  item = get_from_queue()
+
+  with sentry_sdk.start_span(op="task", transaction=item.get_transaction()):
+      # process_item may create more spans internally (see next examples)
+      process_item(item)
+```
+
+**Adding More Spans to the Transaction**
+
+The next example is called in the `process_item` function from the code snippet above. Our SDK can determine if there is a current open `transaction` and add all newly created [spans]({%- link _documentation/apm/apm-glossary.md -%}#span) as child operations to the `transaction`. Keep in mind; each individual span also needs to be finished; otherwise, it will not show up in the `transaction`.
+
+You can choose the value of `op` and `description`.
+
+```python
+import sentry_sdk
+
+with sentry_sdk.start_span(op="http", description="GET /") as span:
+    response = my_custom_http_library.request("GET", "/")
+    span.set_tag("http.status_code", response.status_code)
+    span.set_data("http.foobarsessionid", get_foobar_sessionid())
+```
+
+### JavaScript
+
+To access our APM features, you need to use a beta release.
+
+```bash
+$ npm install @sentry/browser
+$ npm install @sentry/apm
+```
+
+**Sending Traces**
+
+Tracing resides in the `@sentry/apm` package. You can add it to your `Sentry.init` call:
+
+```javascript
+import * as Sentry from '@sentry/browser';
+import { Integrations as ApmIntegrations } from '@sentry/apm';
+
+Sentry.init({
+  dsn: '___PUBLIC_DSN___',
+  integrations: [
+    new ApmIntegrations.Tracing(),
+  ],
+});
+```
+
+You can pass a lot of different options to tracing, but it comes with reasonable defaults out of the box. You'll receive:
+
+- A [transaction]({%- link _documentation/apm/apm-glossary.md -%}#transaction) for every page load
+- All XHR/fetch requests as spans
+
+**Using Tracing Integration for Manual Tracing**
+
+Tracing will create a [transaction]({%- link _documentation/apm/apm-glossary.md -%}#transaction) on page load by default; all [spans]({%- link _documentation/apm/apm-glossary.md -%}#span) that are started will be attached to it. Also, tracing will finish the transaction after the default timeout of 500ms of inactivity. Inactivity is if there are no pending XHR/fetch requests. If you want to extend it by adding spans to the transaction, here is an example of how you could profile a React component:
+
+```javascript
+// This line starts an activity (and creates a span).
+// As long as you don't pop the activity, the transaction will not be finished and therefore not sent to Sentry.
+// If you do not want to create a span out of an activity, just don't provide the second arg.
+const activity = ApmIntegrations.Tracing.pushActivity(displayName, {
```

Comment:
@HazAT do we need docs to explain this concept of "activity" further?

Reply:
^ @HazAT ?

Reply:
@MimiDumpling I think it should be fine omitting this for this PR. Would it be fine if we create an Asana task to track this?